core/commands/add.go

```diff
@@ -188,6 +188,8 @@ You can now check what blocks have been created by:
 			})
 		}
 
+		events := make(chan interface{}, adderOutChanSize)
```

Comment:
Sharness complains, probably because we still have buffered events when we get status on `errCh`